docs/getting-started/getting-started.md

```diff
@@ -2,22 +2,35 @@
 
 You can either get started with K8sGPT in your own environment, the details are provided below or you can use our [Playground example on Killrcoda](../tutorials/playground.md).
 
+!!! tip
+    Please only use K8sGPT on environments where you are authorized to modify Kubernetes resources.
+
 ## Prerequisites
 
 1. Ensure `k8sgpt` is installed correctly on your environment by following the [installation](./installation.md).
-2. You need to be connected to any Kubernetes cluster.
+2. You need to be connected to any Kubernetes cluster. Below is the documentation for setting up a new KinD Kubernetes cluster. However, make sure that kubectl is already installed.
 
 ### Setting up a Kubernetes cluster
 
 To give `k8sgpt` a try, set up a basic Kubernetes cluster, such as KinD or Minikube (if you are not connected to any other cluster).
 
-!!! tip
-    Please only use K8sGPT on environments where you are authorized to modify Kubernetes resources.
-
 - The [KinD documentation](https://kind.sigs.k8s.io/docs/user/quick-start/) provides several installation options to set up a local cluster with two commands.
 
 - The [Minikube documentation](https://minikube.sigs.k8s.io/docs/start/) covers different Operating Systems and Architectures to set up a local Kubernetes cluster running on a Container or Virtual Machine.
 
+
+**Creating a KinD Kubernetes Cluster**
+
+Install KinD first:
+```bash
+brew install kind
+```
+
+Create a new Kubernetes cluster:
+```bash
+kind create cluster --name k8sgpt-demo
+```
+
```

Comment:
Will it look good if we first refer a user to the KinD and Minikube documentation and then also list out the steps for installing a KinD cluster? @AnaisUrlichs

Reply:
We have to use one cluster as an example and I have found KinD most accessible to the other options -- we could use another cluster as an example environment to use K8sGPT in